tests/e2e/specs/subscriptions/shopper/shopper-subscriptions-purchase-multiple-subscriptions.spec.js

```diff
@@ -74,20 +65,32 @@ describeif( RUN_SUBSCRIPTIONS_TESTS )(
 			await page.waitForNavigation( { waitUntil: 'networkidle0' } );
 
 			// Ensure 'Subscription totals' section lists the subscription products with the correct price.
-			const subTotalsRows = await page.$$( 'tr.order_item' );
+			const subTotalsRows = await page.$$(
+				'.shop_table order_details tbody tr.order_item'
```

Comment:
Moving all selectors to constants also makes sense to me to have descriptive names for them.

Reply:
See https://github.com/Automattic/woocommerce-payments/pull/4657#discussion_r959864703

Reply:
Added pre-defined selectors